File: Java/1051.java

import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);

        double salary, renda;

        salary = input.nextDouble();

        if (salary <= 2000.00) {
            System.out.println("Isento");
        }

        else if (salary >= 2000.01 && salary <= 3000.00) {
            renda = (salary - 2000) * 0.08;
            System.out.printf("R$ %.2f\n", renda);
        }

        else if (salary >= 3000.01 && salary <= 4500.00) {
            renda = (salary - 3000) * 0.18 + (1000 * 0.08);
            System.out.printf("R$ %.2f\n", renda);
        }

        else {
            renda = (salary - 4500) * 0.28 + (1500 * 0.18) + (1000 * 0.08);
            System.out.printf("R$ %.2f\n", renda);
        }

    }
}

File: Python/1051.py
renda = float (input())

if renda <= 2000:
    print ("Isento")

elif renda > 2000 and renda < 3001:
    liquid = (renda - 2000) * 0.08
    print (f"R$ {liquid:.2f}")

elif renda > 3000 and renda < 4501:
    liquid = (renda - 3000) * 0.18 + (1000*0.08)
    print (f"R$ {liquid:.2f}")

else:
    liquid = (renda - 4500) * 0.28 + (1500 *0.18) + (1000* 0.08)
    print (f"R$ {liquid:.2f}")

